requirements.txt

```diff
@@ -39,7 +39,7 @@ pysaml2==5.1.0
 SQLAlchemy==1.3.18
 psycopg2==2.8.5
 mysqlclient==1.4.6
-google-cloud-bigquery==1.26.0
+google-cloud-bigquery[pandas]==2.4.0
```

Comment:
why not use [google-cloud-bigquery](https://pypi.org/project/google-cloud-bigquery/) directly? Newest is on 2.5.0

Reply:
`pandas` is a now an '[extra](https://stackoverflow.com/a/52475030/3708872)' for google-cloud-bigquery package. If you don't include it like this there is no support for Pandas, which we're using in the cron.

Reply:
It's a way of creating a package now and not requiring unnecessary packages that you may not need support for. Kind of cool really.